### ISSUE TITLE
SendToConversation results in NotFound response

### ISSUE THREAD

**Issue by ukphillips** [Microsoft/BotBuilder#4689]
## Bot Info
* SDK Platform: .NET
* SDK Version: 3.15.2.2
* Active Channels: Direct Line
* Deployment Environment: local development with Emulator

## Issue Description
I am trying to handle authentication out of band due to limitations with the identity provider. I want to send the user to a separate webpage for authentication, but pass along the conversation id from the bot. Once the user successfully logs in, I want to set data on the conversation (token, other user information), and then send an event activity to the bot service so the chat can know the user is authenticated. (Yes this is a poor man's oauth flow)

When the event activity is sent I get the following response:

![image](https://user-images.githubusercontent.com/18452981/40852884-05eb0818-659a-11e8-81dd-13302851af23.png)

I am modifying a lot of the samples code so some of the namespaces will look familiar.

## Code Example
 ConnectorClient client = new ConnectorClient(new Uri([BotServiceUrl]), ConfigurationManager.AppSettings["MicrosoftAppId"], ConfigurationManager.AppSettings["MicrosoftAppPassword"]);
            
client.Conversations.SendToConversation([ConvoId], new Activity() { ReplyToId = [ConvoId], ChannelId = [ChannelId],  Type = "event", Text = "LoggedIn" });

The botserviceurl is sent over from bot service in the query string and comes from the activity.ServiceUrl
BotServiceUrl (activity.ServiceUrl), ConvoId (activity.Conversation.Id), ChannelId (activity.ChannelId) are all sent over in the query from the bot service

## Reproduction Steps
1.
2.
3.

## Expected Behavior
To be able to send an event back to the originating conversation

## Actual Results
Error is returned

**Comment by ukphillips** [comment]
I also attempted to solve this with the DirectLineClient with no luck. I replaced the SendToConversation (which I noticed I incorrectly populated the ReplyToId field, this should be a user to reply to) with:
```
var dlClient = new DirectLineClient(ConfigurationManager.AppSettings["DirectLineSecret"]);
var post1 = await dlClient.Conversations.PostActivityAsync(convoId, new Connector.DirectLine.Activity() { Type = Connector.DirectLine.ActivityTypes.Event, Text = "LoggedIn" }).ConfigureAwait(false);
```
This resulted in nothing being sent over to my bot (according to the receiving end), and returning a null value to post1, which makes me think there is an error occurring. I actually tried to take the conversation id and directly post it to the directline REST endpoint:

https://directline.botframework.com/v3/directline/conversations/67c044311cca42feb32ae2c4e869300d/activities

which returned a 404 from postman: 
```
{
  "error": {
    "code": "BadArgument",
    "message": "Unknown conversation"
  }
}
```

I also tried this with a blank echo bot template that I spun up in azure, and modifed the code to echo the conversation id, I accessed it through the test chat in the portal and took the conversation id it returned and put this in the REST endpoint with the same result. At a minimum I would have expected this final case to work. Are my expectations about how the direct line bot communication should work incorrect?
